Automattic/woocommerce-payments
# 9341-warning-due-to-unhandled-possible-return-value

Fixes #9341

#### Changes proposed in this Pull Request

When going through checkout without a payment method, the plugin throws the following warning:
```
PHP Warning:  Attempt to read property "id" on bool in /home/public_html/wp-content/plugins/woocommerce-payments/includes/class-woopay-tracker.php on line 546
```

#### Testing instructions

* create a coupon that has a big value, and will enable free shipping on checkout
* add a product to checkout, and apply the coupon created
* when completing the checkout the total sum to pay should be 0

-------------------


**Post merge**
